src/components/CorporateCampaign/CampaignProgressBar.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+	<div
+		class="tw-rounded tw-bg-brand-50 tw-w-full tw-py-1 tw-px-1 tw-mb-2 tw-mt-2"
+	>
+		<template v-if="promoAmount">
+			<h4 class="tw-mt-2 tw-mb-1 tw-px-4">
+				Let's Get Started
+			</h4>
+			<h3 class="tw-mb-2 tw-px-4">
+				You have ${{ promoAmount | numeral }} in credit from {{ promoName }}
+			</h3>
+			<kv-progress-bar
+				class="tw-mb-2"
+				aria-label="Percentage of credit amount"
+				:max="promoAmount.toString()"
+				:value="(0).toString()"
```

Comment:
Are these static bars? It looks this bar will always just be gray with this value.

![image](https://user-images.githubusercontent.com/16867161/220799749-b6f8bd26-e509-4d5b-84db-acd0abffe002.png)